ui/js/values.js

```diff
@@ -108,6 +123,8 @@ treeherder.value("thPlatformMap", {
     "horizon-win64": "Horizon Windows x64",
 
     "gecko-decision": "Gecko Decision Task",
+    "lint": "Linting",
+    "release-mozilla-release-": "Balrog Publishing",
```

Comment:
From what I can see of what's there now[1], yes.

1. https://treeherder.mozilla.org/#/jobs?repo=mozilla-release&filter-searchStr=%3F&selectedJob=120979965